modules/sms_user/sms_user.module

```diff
@@ -61,6 +64,9 @@ function sms_user_send($uid, $message) {
  *
  * @return string|bool
  *   The user's verified number, or false if the user does not have access.
+ *
+ * @deprecated
+ *   See \Drupal::service('sms.phone_number')->getPhoneNumbers()
```

Comment:
This should document the new preferred way of doing the same thing.
 Add a line note